### DISCUSSION TITLE
Support for plaintext view

### DISCUSSION THREAD

**Discussion by RoryCrispin** [grafana/clickhouse-datasource#408] [Ideas]
Some queries executed against the DB return plain text, for example
'SHOW CREATE default.uk_price_paid' 

None of the existing Formats (Auto, Table, Time Series, Logs, Trace) support displaying the output of this data. An option to use plaintext would be helpful.

**Comment by mshustov** [comment]
You can use SQL editor + Table view for this
![2023-06-09_09-41-27](https://github.com/grafana/clickhouse-datasource/assets/3198181/707b612e-5af4-4b45-b363-1f6af4012863)
